migrations/versions/20181224_user_measure_table_rename_user_page_table_to_user_measure.py

```diff
@@ -0,0 +1,113 @@
+"""Rename `user_page` table to `user_measure`
+
+Revision ID: 20181224_user_measure_table
+Revises: 20181220_populate_measures
+Create Date: 2018-12-24 09:52:07.668219
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "20181224_user_measure_table"
+down_revision = "20181220_populate_measures"
+branch_labels = None
+depends_on = None
+
+max_measure_version_temp_table = """
+CREATE TEMPORARY TABLE max_measure_version
```

Comment:
Just to check... we're only picking `MAX` here as a shorthand to get one row per measure, rather than because the `measure_version` would ever differ from one version to the next?

Reply:
Yep